specs/latest/1.0/index.html

```diff
@@ -3316,43 +3323,66 @@ <h4>Uniforms and attributes</h4>
 
 <!-- ======================================================================================================= -->
 
-    <h4><a name="WRITING_TO_THE_DRAWING_BUFFER">Writing to the drawing buffer</a></h4>
+    <h4><a id=DRAW_OPERATIONS>Draw Operations</a></h4>
 
     <p>
-        OpenGL ES 2.0 has 3 calls which can render to the drawing buffer: <code>clear</code>,
-        <code>drawArrays</code> and <code>drawElements</code>. Furthermore rendering can
+        OpenGL ES 2.0 has several calls which are allowed to
+        write to the currently bound (draw) framebuffer.
+        WebGL categorizes all such calls as <a href=#DRAW_OPERATIONS>[Draw Operations]</a>.
+    </p>
+    <p>
+        Furthermore, rendering can
         be directed to the drawing buffer or to a Framebuffer object. When rendering is
-        directed to the drawing buffer, making any of the 3 rendering calls shall
+        directed to the drawing buffer, making any of the rendering calls shall
         cause the drawing buffer to be presented to the HTML page compositor at the start
         of the next compositing operation.
     </p>
-
     <p>
-        If any one of these 3 calls attempts to draw to a missing attachment of a complete framebuffer,
+        These include, but (due to additions in extensions or in WebGL 2) are not limited to:
+        <ul>
+            <li><code>clear</code></li>
+            <li><code>drawArrays</code></li>
+            <li><code>drawElements</code></li>
+        </ul>
+        For instance,
+        <a href="https://registry.khronos.org/webgl/extensions/ANGLE_instanced_arrays/">ANGLE_instanced_arrays</a>
```

Comment:
Consider using a relative path since it's a part of the same repo.

Reply:
Most urls in that document are absolute also. With relative paths, it would become tricky when we snapshot the spec, because it will move in the directory heirarchy:
```
$ find specs -name index.html
specs/1.0.0/index.html
specs/1.0.1/index.html
specs/1.0.2/index.html
specs/1.0.3/index.html
specs/2.0.0/index.html
specs/latest/1.0/index.html
specs/latest/2.0/index.html
specs/latest/index.html
```
We could of course make changes such that it could maintain the same depth when snapshotted, but that's part of the context.